ethereum/blockcreation/src/main/java/org/hyperledger/besu/ethereum/blockcreation/txselection/selectors/PriceTransactionSelector.java

```diff
@@ -71,30 +71,27 @@ public TransactionSelectionResult evaluateTransactionPostProcessing(
    */
   private boolean transactionCurrentPriceBelowMin(final PendingTransaction pendingTransaction) {
     final Transaction transaction = pendingTransaction.getTransaction();
-    // Here we only care about EIP1159 since for Frontier and local transactions the checks
-    // that we do when accepting them in the pool are enough
-    if (transaction.getType().supports1559FeeMarket() && !pendingTransaction.hasPriority()) {
```

Comment:
Just double checking, did you mean to drop the first condition of this if?

Reply:
Yes, because now the `minGasPrice` could change at runtime, so the check done in the txpool is no more enough, and we need to redo it here for every tx